lib/LedgerSMB/Tax.pm

```diff
@@ -47,7 +47,7 @@ sub init_taxes {
     my @accounts = split / /, $taxaccounts;
     if ( defined $taxaccounts2 ) {
         #my @tmpaccounts = @accounts;#unused var
-        $#accounts = -1;# empty @accounts,@accounts=();
+        @accounts=(); # empty @accounts
```

Comment:
Running a quick test from the command line shows that the original works.

```bash
$ perl -MData::Dumper -e '@a=(1,2,3); print Dumper(\@a); $#a=-1; print Dumper(\@a);'
$VAR1 = [
          1,
          2,
          3
        ];
$VAR1 = [];
```

I have no idea why anybody would want to set the index of the last element. I'm all for the cleanup that was applied here.